flax/linen/module.py

```diff
@@ -1462,6 +1462,48 @@ def __call__(self, x):
     self.scope.put_variable(col, name, xs)
     return True
 
+  def perturb(self, col: str, name: str, value: T) -> T:
```

Comment:
I wonder if we should encourage a default (but still configurable) collection name "perturbations" or some such here...

Reply:
Fixed both! Made "collections" a keyword arg with default, and modified the docstring.